README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly, update your README to better represent the project. Please use the MV template.

Reply:
Thank a lot for your review 😄